index.js

```diff
@@ -9,3 +9,36 @@ const hello = (name) => {
 
 oldHello('world');
 hello('es6');
+
+
+$(document).ready(function(){
+  listContacts()
+})
+
+var contacts = [
+  "Andy Mention",
+  "Emily Davis",
+  "Eric Fransen",
+  "Jessica Goulding",
+  "Jonmichael Chambers",
+  "Marc Garreau",
+  "Tan Doan",
+  "Alan Smith",
+  "Allison Larson",
+  "Andrw Watkins",
+  "Chad Brading",
+  "Corey Davis",
+  "Gustavo Villagrana",
+  "Hilary Denton",
+  "Horacio Chavez",
+  "Tim Proctor",
+  "Will Faurot"
+]
+
+function listContacts() {
+  contacts.forEach(function(contact) {
```

Comment:
This looks good. 👍  The only things I would look into are ES6 related.
- using `map()` instead of `forEach()`
- using `arrow functions`
- using `const` for naming this function